src/load_data.jl

```diff
@@ -53,6 +53,80 @@ function load_data(filename::AbstractString)
 
 end
 
+function load_auspex_data(filename::AbstractString)
+    # FILE STRUCTURE:
+    # file.hdf5:
+    #    group1/
+    #       data/
+    #          axis_1 (contains all tuples visited on this axis)
+    #          axis_2 (contains all tuples visited on this axis)
+    #          quantity_1 (the measured values)
+    #          quantity_2 (the measured values)
+    #       descriptor (contains references to the axes datasets below)
+    #       axis_1 (contains the values of the axis — see note #1)
+    #       axis_2 (contains the values of the axis - see note #1)
+    #    group2/
+    #       ... same structure as above ...
+    #
+    # Note #1: Any axes referenced by the descriptor can themselves contain
+    #          references to multiple subaxes. This indicates that the axis
+    #          is "unstructured", i.e. that multiple coordinates where changed
+    #          simultaneously.
+
+    datasets    = Dict{String, Dict{String, Vector{Any}}}()
```

Comment:
Does this force keys of the internal `Dict`s to have `Vector{Any}` data type? If so, it might be better to declare this to be `Dict{String, Dict{String, Any}}`